website/docs/topics/feature-flags/feature-flag-best-practices.md

```diff
@@ -1,339 +1,188 @@
 ---
 title: "Feature flags: Best practices for building and scaling"
```

Comment:
Could this just be:

```suggestion
title: "Best practices for building and scaling feature flags"
```

Reply:
For the page title (the thing that is in the browser tab), it is important to keep "Feature flags: Best practices for building and scaling". The H1 (on page) can be what you suggested though.